packages/tkg/bundle/config/values.yaml

```diff
@@ -86,6 +86,10 @@ tkrSourceControllerPackage:
     skipVerifyRegistryCert: false
     initialDiscoverFrequency: 60
     continuousDiscoverFrequency: 600
+    tkgProxyCaCert:
+    tkgCustomImageRepositoryCaCert:
```

Comment:
Thought: Instead of having two variables `tkgProxyCaCert`, `tkgCustomImageRepositoryCaCert`, we can have single variable and have the logic to prioritize/choose the right CA can be done in CLI

Reply:
makes sense. updated